interface/reports/patient_edu_web_lookup.php

```diff
@@ -43,10 +33,7 @@
 
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-3-2/index.js"></script>
-
```

Comment:
lets stick a title here for this page (this is what then gets used on the tab title). maybe something like:
``````
<title><?php echo xlt('Patient Education'); ?></title>